File: src/dotnet/ArdsTools/Ards.NetCore.Entity/Models/QslVia.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.NetCore.Entity.Models
{
    [Table("qsl_via", Schema = "adif")]
    public partial class QslVia
    {
        [Column("id")]
        public int Id { get; set; }
        [Required]
        [Column("via")]
        [StringLength(1)]
        public string Via { get; set; }
        [Required]
        [Column("description")]
        [StringLength(20)]
        public string Description { get; set; }
        [Column("import_only")]
        public bool ImportOnly { get; set; }
    }
}

File: src/java/ards-adif-api/src/main/java/com/ards/adif/api/model/QslVia.java
package com.ards.adif.api.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the qsl_via database table.
 * 
 */
@Entity
@Table(name="qsl_via")
@NamedQuery(name="QslVia.findAll", query="SELECT q FROM QslVia q")
public class QslVia implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(nullable=false, length=20)
	private String description;

	@Column(name="import_only", nullable=false)
	private Boolean importOnly;

	@Column(nullable=false, length=1)
	private String via;

	public QslVia() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public String getDescription() {
		return this.description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public Boolean getImportOnly() {
		return this.importOnly;
	}

	public void setImportOnly(Boolean importOnly) {
		this.importOnly = importOnly;
	}

	public String getVia() {
		return this.via;
	}

	public void setVia(String via) {
		this.via = via;
	}

}